dev-requirements.txt

```diff
@@ -23,6 +23,9 @@ pytest-cov>=2.7.0; python_version == '2.7' or python_version >= '3.5'
 git+https://github.com/andy-maier/coveralls-python.git@andy/add-py27#egg=coveralls; python_version == '2.7'
 coveralls>=2.1.2,<3.0.0; python_version >= '3.5'
 
+# version 8.11 requires python 3.6, See issue #2796
+more-itertools<=8.10.0; python_version <= '3.5'
```

Comment:
I think we should be more brave and just exclude more-itertools 8.11.0 instead of pinning it, in expectation of them fixing that.

In addition, I suggest using a minimum version. In other projects, I used 4.0.0 as the minimum version.

This then results in:
```
more-itertools>=4.0.0,!=8.11.0; python_version < '3.6'
more-itertools>=4.0.0; python_version >= '3.6'
```

Reply:
DONE